test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityControllerSpec.scala

```diff
@@ -4384,6 +4607,59 @@ class YearToDateLiabilityControllerSpec
 
       "redirect to the check your answers page" when {
 
+        "the user is on a further return journey where a previous year to date value is available" in {
```

Comment:
it'd be good to have a success test case for the further return enter tax due page as well